server/src-lib/Hasura/Server/App.hs

```diff
@@ -730,8 +755,7 @@ httpApp corsCfg serverCtx enableConsole consoleAssetsDir enableTelemetry = do
       Spock.post "v1alpha1/graphql" $ spockAction GH.encodeGQErr id $
         mkPostHandler $ mkAPIRespHandler $ v1Alpha1GQHandler E.QueryHasura
 
-      Spock.post "v1/graphql" $ spockAction GH.encodeGQErr allMod200 $
-        mkPostHandler $ mkAPIRespHandler v1GQHandler
+      runGQLApiHandler serverCtx 
```

Comment:
Maybe adding the following comment will help the future reader understand what's going on?

```suggestion
      -- See 'MonadGQLApiHandler'
      runGQLApiHandler serverCtx 
```